pyproject.toml

```diff
@@ -14,8 +14,8 @@ classifiers = [
 ]
 
 dependencies = [
-  "beartype==0.14.0",
-  "deepdiff==6.3.0",
+  "beartype==0.18.5",
+  "deepdiff==7.0.1",
```

Comment:
Would probably be best to pin these to a range instead of specific versions. That way users of mystique can take advantage of dependency updates without you having to make a new release.

Reply:
Yeah, I am not yet using a locking package manager here. So these are forced values using pip.
I think I will remove the 'dependencies' section entirely from pyproject since nothing is using it.